darwin/DarwinProcess.c

```diff
@@ -398,6 +408,9 @@ void DarwinProcess_setFromLibprocPidinfo(DarwinProcess* proc, DarwinProcessTable
    dpt->super.userlandThreads += pti.pti_threadnum; /*pti.pti_threads_user;*/
    dpt->super.totalTasks += pti.pti_threadnum;
    dpt->super.runningTasks += pti.pti_numrunning;
+#else
+   proc->taskAccess = false;
+#endif
```

Comment:
Try to fill all the other fields here too; and be it with sensible "defaults" like "no time passed" for "one task"/"none running". Mark a todo to implement these with older APIs as necessary.